index.js

```diff
@@ -0,0 +1,15 @@
+var express = require( "express" );
+var app;
```

Comment:
Why don't you just declare this one where you call `express()`? Same thing with the other variables.